service/terraform-plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/terraform/configuration/TerraformProperties.java

```diff
@@ -0,0 +1,12 @@
+package com.solace.maas.ep.event.management.agent.plugin.terraform.configuration;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+@Data
+public class TerraformProperties {
+    @Value("${plugins.terraform.workingDirectoryRoot:/${HOME}/tfconfig}")
```

Comment:
This is an unorthodox way of setting properties. @Value is not the recommended way to populate config. In this case I wanted a convenient way to set a default value for this value without requiring a change to the application.yml file. Alternatively, we could add a backend story to EP to create a new `plugins.terraform.workingDirectoryRoot` property to the generated application.ylm file.